test/helpers/hosts_and_hostgroups_helper_test.rb

```diff
@@ -173,6 +173,7 @@ def setup
 end
 
 class HostsAndHostGroupsHelperKickstartRepositoryIDTest < HostsAndHostGroupsHelperTestBase
+  include Katello::KatelloUrlsHelper
```

Comment:
I think this tests should be extracted to separate helper test, something like `katello_urls_helper_test.rb` I was pointing to this helper to take inspiration on how to build it.

Reply:
I actually thought that it does belong to this place. but I'll create a new one.